spec/models/worker_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Worker, type: :model do
+	let(:test_worker) { create(:worker) }
+
+	describe "validations" do
+		subject { test_worker }
+
+		it { is_expected.to validate_presence_of(:first_name) }
+		it { is_expected.to validate_presence_of(:last_name) }
+		it { is_expected.to validate_presence_of(:worker_city) }
+		it { is_expected.to validate_presence_of(:worker_state) }
+	end
+
+	describe "worker_full_name method" do
```

Comment:
Hey Cam! Awesome work!

For the `worker_full_name` spec, I think it is better to use the `it "should do something" do ... end` syntax.

One main reason is so that, if the test fails, the failure description makes sense.

The way you have it now, I broke the test and ran it and got the following output:

```ruby
Failures:

  1) Worker worker_full_name method is expected to eq "JonSnow"
     Failure/Error: it {expect(test_worker.worker_full_name).to eq("JonSnow")}
     
       expected: "JonSnow"
            got: "Jon Snow"
     
       (compared using ==)
     # ./spec/models/worker_spec.rb:18:in `block (3 levels) in <top (required)>'

Finished in 0.20751 seconds (files took 1.39 seconds to load)
5 examples, 1 failure

Failed examples:

rspec ./spec/models/worker_spec.rb:18 # Worker worker_full_name method is expected to eq "JonSnow"
```

The line ` Worker worker_full_name method is expected to eq "JonSnow"` does not actually describe what was expected from the spec.

I think a better syntax is:

```ruby
describe "worker_full_name method" do
    it "should return the first and last name of the worker, separated by a space" do
        expect(test_worker.worker_full_name).to eq("Jon Snow")
    end
end
```

A failing spec would then yield this more descriptive lead-in:

`Worker worker_full_name method should return the first and last name of the worker, separated by a space`

Let me know what you think 😄

Reply:
Thanks, @msarit! Have updated the test descriptions to be more descriptive. It helps make more sense of test failures and feels almost like in-depth documentation for the methods. Will embrace this for future tests, thank you!